src/actions/index.js

```diff
@@ -1,4 +1,7 @@
 import request from 'superagent'
+const env = process.env.NODE_ENV || 'development'
+const url = env === 'production' ? 'http://s2pid-kweschinz.herokuapp.com/'
```

Comment:
Please use https for the live server: all herokuapp.com subdomains are SSL by default.